.travis.yml

```diff
@@ -10,54 +10,64 @@ stages:
 jobs:
   include:
     - stage: test
+      name: osx
       os: osx
       before_install:
         - SKIP=`.travis/check-mergify-merge` && if [[ "$SKIP" = "skip" ]]; then exit 0; fi
         - SKIP=`.travis/check-change '^(docs|spec)/'` && if [[ "$SKIP" = "skip" ]]; then exit 0; fi
-      install:
         - SKIP=`.travis/check-change '^test/'` && if [[ "$SKIP" != "skip" ]]; then RUN_UNIT_TEST=1; fi
-        - test $RUN_UNIT_TEST && rustup toolchain install nightly-2018-12-06 || true
-        - test $RUN_UNIT_TEST && rustup component add rustfmt-preview --toolchain nightly-2018-12-06 || true
-        - test $RUN_UNIT_TEST && rustup component add clippy-preview --toolchain nightly-2018-12-06 || true
+      install:
         - nvm install 10
         - nvm use 10
         - npm install -g yarn
       before_script:
         - cargo fetch --verbose
       script:
-        - (test $RUN_UNIT_TEST && cargo +nightly-2018-12-06 fmt -- --check || true) || exit 1
-        - (test $RUN_UNIT_TEST && cargo +nightly-2018-12-06 clippy --all --all-targets -- -D warnings || true) || exit 1
         - (test $RUN_UNIT_TEST && RUST_BACKTRACE=1 cargo test --verbose --all || true) || exit 1
```

Comment:
`if [[ $RUN_UNIT_TEST ]]; then RUST_BACKTRACE=1 cargo test --verbose --all; fi`
is less cryptic.